sui/deploy-test.js

```diff
@@ -42,8 +46,10 @@ async function processCommand(config, chain, options) {
 
     await broadcast(client, keypair, tx);
 
+    const channelId = await getChannelId(client, singleton.objectId);
+
     chain.contracts.test.address = published.packageId;
-    chain.contracts.test.objects = { singleton: singleton.objectId };
+    chain.contracts.test.objects = { singleton: singleton.objectId, channelId };
```

Comment:
submit register_transaction after deployment

Reply:
not sure if i understand you correctly, i think it's already submitted register_transaction after the deployment?